tests/usability/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test command abbreviation"
+        rlRun -s "tmt run pl disc provi -h connect 2>&1" "1-255"
+        rlAssertNotGrep "Unsupported discover method 'connect'" $rlRun_LOG
+        rlAssertGrep "Provide a host name or an ip address to connect" $rlRun_LOG
```

Comment:
this line is no longer needed. The `tmt run` has to pass.

Reply:
Removed.